src/openrct2/ride/Ride.h

```diff
@@ -992,7 +992,8 @@ struct rct_ride_properties
 #define TURN_MASK_3_ELEMENTS 0x0700
 #define TURN_MASK_4_PLUS_ELEMENTS 0xF800
 
-#define CONSTRUCTION_LIFT_HILL_SELECTED 1
+constexpr uint32_t CONSTRUCTION_LIFT_HILL_SELECTED = 1 << 0;
+constexpr uint32_t CONSTRUCTION_INVERTED_TRACK_SELECTED = 1 << 1;
```

Comment:
Are they really unsigned? Because on all places they're being compared to `int32_t`

Reply:
I think we tend to use unsigned for flags for edge cases at 1 << 31

Reply:
I think using unsigned for flags makes most sense. After all, there is no such thing as a "negative flag".